openlibrary/core/civicrm.py

```diff
@@ -80,6 +80,5 @@ def get_sponsorships_by_contact_id(contact_id=None, isbn=None):
         "context": t.pop(CIVI_CONTEXT),
         "receive_date": t.pop("receive_date"),
         "total_amount": t.pop("total_amount"),
-        "contact_id": t.pop("contact_id"),
-        "contribution_status": t.pop("contribution_status")
+        "contact_id": "270430"
```

Comment:
This is not the correct place to make the change.  There is a method named `fake_civi` in upstream/account.py that mocks API responses for local environments.  That is what needs to be modified.

https://github.com/internetarchive/openlibrary/blob/76f28b83eec379945fff6e2d66a6138adfad180c/openlibrary/plugins/upstream/account.py#L842-L861

Reply:
When you update this, `Completed` should be a good value for `contribution_status`.

Reply:
I have made the changes.